x-pack/legacy/plugins/apm/public/components/shared/MetadataTable/SpanMetadata/sections.ts

```diff
@@ -15,10 +15,10 @@ import {
 } from '../sections';
 
 export const SPAN_METADATA_SECTIONS: Section[] = [
+  LABELS,
   SPAN,
-  AGENT,
-  SERVICE,
   TRANSACTION,
-  LABELS,
-  TRACE
+  TRACE,
+  SERVICE,
+  AGENT
```

Comment:
Would it make sense to order them alphabetically - with the exception of labels?
For instance: we have another issue open to add `exception.attributes`. If the list is not alphabetically ordered, it'll just be added somewhere arbitrarily, and then the order will become a little random again.

Reply:
We've always tried to curate the order because of relative importance in what data you're interested in. I agree that at some point it becomes so large that the curated order is lost and it might be better with an alphabetical list. I'm not sure which is better. I can raise this with the agent devs and do a poll.

Reply:
Okay, so when adding new sections we should remember to have a chat about where that section belongs.

Reply:
I think that applies to any time we want to add new data to the UI it kind of needs to happen with some sort of intention, agreed.